skyline-vscode/react-ui/src/utils.js

```diff
@@ -159,15 +187,38 @@ export function energy_data(currentTotal){
       `${parseFloat(Number(carbon_emission_tons*1000).toFixed(2))} kg`: 
       `${parseFloat(Number(carbon_emission_tons).toFixed(2))} Metric Tons`;
   const miles = unitScale(carbon_emission_tons * ENERGY_CONVERSION_UNITS['miles'],'generic');
-  const household = carbon_emission_tons * ENERGY_CONVERSION_UNITS['household'];
+  const household = timeFormatter(carbon_emission_tons * ENERGY_CONVERSION_UNITS['household']);
   const phone = unitScale(carbon_emission_tons * ENERGY_CONVERSION_UNITS['phone'],'generic');
 
   return {
     kwh: parseFloat(Number(kwh).toFixed(2)),
     carbon: carbon_unit,
     miles: `${miles.val} ${miles.scale}`,
-    household: parseFloat(Number(household).toFixed(2)),
+    household: household,
     phone: `${phone.val} ${phone.scale}`
   }
 
+}
+
+export function calculate_training_time(numIterations, instance){
+  return numIterations * instance.x / 3.6e6 / instance.info.ngpus;
+}
+
+export function numberFormat(num){
+  const formatter = new Intl.NumberFormat("en-US", {
+    notation: "compact",
+    compactDisplay: "long",
+  });
+  // 988M
```

Comment:
what is this comment for?

Reply:
I used as a guide on how the format should look like, I deleted it now